lib/src/chat_view.dart

```diff
@@ -452,117 +422,144 @@ class DashChatState extends State<DashChat> {
+                                    onReply: widget.onQuickReply,
+                                    quickReplyBuilder: widget.quickReplyBuilder,
+                                    quickReplyStyle: widget.quickReplyStyle,
+                                    quickReplyTextStyle:
+                                        widget.quickReplyTextStyle,
+                                  ),
+                                )
+                                .toList(),
+                        ],
+                      ),
+                    ),
+                  if (widget.chatFooterBuilder != null)
+                    widget.chatFooterBuilder(),
+                  if (!widget.readOnly)
+                    ChatInputToolbar(
+                        key: inputKey,
+                        sendOnEnter: widget.sendOnEnter,
+                        textInputAction: widget.textInputAction,
+                        inputToolbarPadding: widget.inputToolbarPadding,
+                        textDirection: widget.inputTextDirection,
+                        inputToolbarMargin: widget.inputToolbarMargin,
+                        showTraillingBeforeSend: widget.showTraillingBeforeSend,
+                        inputMaxLines: widget.inputMaxLines,
+                        controller: textController,
+                        inputDecoration: widget.inputDecoration,
+                        textCapitalization: widget.textCapitalization,
+                        onSend: widget.onSend,
+                        user: widget.user,
+                        messageIdGenerator: widget.messageIdGenerator,
+                        maxInputLength: widget.maxInputLength,
+                        sendButtonBuilder: widget.sendButtonBuilder,
+                        text: widget.text != null ? widget.text : _text,
+                        onTextChange: widget.onTextChange != null
+                            ? widget.onTextChange
+                            : onTextChange,
+                        leading: widget.leading,
+                        trailling: widget.trailing,
+                        inputContainerStyle: widget.inputContainerStyle,
+                        inputTextStyle: widget.inputTextStyle,
+                        inputFooterBuilder: widget.inputFooterBuilder,
+                        inputCursorColor: widget.inputCursorColor,
+                        inputCursorWidth: widget.inputCursorWidth,
+                        showInputCursor: widget.showInputCursor,
+                        alwaysShowSend: widget.alwaysShowSend,
+                        scrollController: widget.scrollController != null
+                            ? widget.scrollController
+                            : scrollController,
+                        focusNode: inputFocusNode,
+                        reverse: widget.inverted)
+                ],
+              ),
+              if (visible)
+                Positioned(
+                  right: 0,
```

Comment:
Currently, this aligns the button to the center of the screen, so I was thinking a new property may be `scrollButtonAlignment` would be a good option, which will be an enum `left, right, center` like so.

Reply:
I think that would be better to have a property to set the exact position, because the bottom position might not be enough if you add something as a footer or more margin/padding to the toolbar, do you think a Map<String, String> with the keys being `right`, `left`, `top`, `bottom` would be ok?

Reply:
Maybe A simple class `ScrollToBottomWidgetStyle` with property `backgroundColor, fillColor, height, width, left, right, bottom` would be better.

Reply:
It would also help with making the package more modular.